docs/adr/0003-server-side-node.md

```diff
@@ -1,30 +1,29 @@
-# 0003. Servcer Side Node Standards
+# 0003. Server Side Node Standards
 
 Date: 2020-01-21
 
 ## Status
 
-Suggested
+Accepted
 
 ## Context
 
 As part of on-going development, it has been noticed (thanks Norris) that the
-tranpilation is not what was expected and we should use a later version of 
+tranpilation is not what was expected and we should use a later version of
 ECMA script to target.
 
 ## Decision
 
-From [here]( https://kangax.github.io/compat-table/es2016plus/#node12_11)
+From [here](https://kangax.github.io/compat-table/es2016plus/#node12_11)
 
-The suggestion is to use node v12.14.1 and target "ES2019" in your `tsconfig.json`
+The suggestion is to use node v12.15.0 and target "ES2019" in your `tsconfig.json` for server components and "ES5" for client components,
```

Comment:
Did we want to mention the motivation for the server change was `alpine` ... i.e. it would be good to keep it in step?

Reply:
this is to keep in sync with node:12.15-alpine docker image